README.md

```diff
@@ -1,2 +1,29 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
 # school_library
-A tool to record what books are in the library and who borrows them
+>A tool to record what books are in the library and who borrows them
+
+## Author
+
+👤 **Akumu Bavon**
+
+- GitHub: [@Bavon101](https://github.com/Bavon101)
+- Twitter: [@akumubavon](https://twitter.com/akumubavon)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/akumu-bavon/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+
+## 📝 License
+
```

Comment:
- [ ] please make sure to use [the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse, including the links (for example to the MIT license). keep in mind that all sections of the template are mandatory except for the "Live Demo", "Run Tests" and "Deployment" sections, these are only mandatory if the specific project you're working on requires it.